rabbit-escape-render/src/rabbitescape/render/Polygon.java

```diff
@@ -0,0 +1,45 @@
+package rabbitescape.render;
+
+import java.util.Vector;
+
+import rabbitescape.engine.util.Util;
+
+public class Polygon
+{
+    public Vector<Integer> x, y;
```

Comment:
i refactored. polygon building is in a separate class so the coordinates can be final.

its in a list of Vertex (which is the only float coordinate class)